runtime/calamari/src/impls.rs

```diff
@@ -28,7 +28,7 @@ pub struct DealWithFees;
 impl OnUnbalanced<NegativeImbalance> for DealWithFees {
 	fn on_unbalanceds<B>(mut fees_then_tips: impl Iterator<Item = NegativeImbalance>) {
 		if let Some(fees) = fees_then_tips.next() {
-			let mut split = fees.ration(0, 100);
+			let mut split = fees.ration(20, 80);
```

Comment:
80 to block authors : 20 to treasury seems more appropriate to me to begin with since we're trying to hype up the collator program

Reply:
Seems 80/20 is recommended.

Reply:
But you deposit 80% fees to block authors, and 20% to treasury.
```rust
Treasury::on_unbalanced(split.0);
Author::on_unbalanced(split.1);
```
polkadot and kusama are opposite，
https://github.com/paritytech/polkadot/blob/master/runtime/common/src/impls.rs#L56

Reply:
I know but maybe 80/20 to collators is going to be more compelling for our collator program, at least for starters ?

Reply:
Also we just made it 100% to block authors last week, and we're planning now to reduce it to 20% now ... It's kind of odd.